### ISSUE TITLE
as_text for Embed objects throws not implemented error

### ISSUE THREAD

**Issue by spnkr** [prismicio/ruby-kit#77]
should return nil or a string or the alt attribute or something. annoying that it errors out.

e.g.
```
module Prismic
  module Fragments
    class Embed < Fragment
      def as_text
        ""
      end
   end
  end
end
```

**Comment by zoltanmiko** [comment]
I completely agree!

**Comment by MarcMcIntosh** [comment]
fixed in v1.8.1
